### ISSUE TITLE
Setup Camera extension

### ISSUE THREAD

**Issue by vidbina** [Suprnovae/abc-totaal-notitieblok#2]
We need to have https://github.com/lwansbrough/react-native-camera set up for iOS and Android

**Comment by vidbina** [comment]
Just add a simple button in the iOS and Android UI's to trigger the camera. We'll move it to it's intended place later. We just need it to work now :wink:

**Comment by zirconias** [comment]
cool,  
what about branching,  features and bug fixes ?   a simple commit ?

**Comment by vidbina** [comment]
Oh yeah. Forgot to mention that. Just start a `setup/camera` branch. You
could implement it for iOS and Android in separate commits 😉

I merged the latest changes to `master` so just start your branch from there.

**Comment by zirconias** [comment]
sounds great,  (y)

**Comment by zirconias** [comment]
we can close the issue now ?

**Comment by vidbina** [comment]
I have to review it first. Only closes when merged to master.

**Comment by vidbina** [comment]
There seems to be an issue with `renderHome()`. I've just ignored it and tried returning the component JSX within the switch-case (instead of the `renderHome()` func) to test how it functions. Discovered that the navigation pushes the Camera views into place, but there seems to be no interface to engage the camera. Furthermore, the Android back button does not pop items from the navigator stack. So I got a few extra items for you to work on, but as far as setting up the extention goes... you've done the job.
